### PR TITLE
chore(CDAP-19229): convert page.level.errors.spec.ts test from cypress to cucumber selenium

### PR DESCRIPTION
**Pull request by de-lan**
# Convert page.level.errors.spec.ts test from cypress to cucumber selenium

## Description
Summary of changes

## PR Type
- [ ] Bug Fix
- [ ] Feature
- [ ] Build Fix
- [x] Testing
- [ ] General Improvement
- [ ] Cherry Pick

## Links
Jira: [CDAP-19229](https://cdap.atlassian.net/browse/CDAP-19229)

## Screenshots
image

### CODE REVIEW

File: src/e2e-test/java/io/cdap/cdap/ui/stepsdesign/PageLevelErrors.java [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+  @When("Open fake namespace pipeline drafts page")
+  public void openFakeNamespacePipelineDraftsPage() {
+    SeleniumDriver.openPage(Constants.BASE_URL + "/cdap/ns/" + fakeNamespace + "/pipelines/drafts");
+    WaitHelper.waitForPageToLoad();
+  }
+
+  @When("Open fake namespace Wrangler page")
+  public void openFakeNamespaceWranglerPage() {
+    SeleniumDriver.openPage(Constants.BASE_URL + "/cdap/ns/" + fakeNamespace + "/wrangler");
```
Comment (radhikav1) [inline src/e2e-test/java/io/cdap/cdap/ui/stepsdesign/PageLevelErrors.java]: Good to define a constant within this class for **"/cdap/ns" + fakeNamespace** and consume as it is used at many places in the class.

File: src/e2e-test/java/io/cdap/cdap/ui/stepsdesign/PageLevelErrors.java [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+  @When("Open invalid pipeline path details page")
+  public void openInvalidPipelinePathDetailsPage() {
+    SeleniumDriver.openPage(Constants.BASE_URL + "/pipelines/ns/default/view/invalidPipelineName");
+    WaitHelper.waitForPageToLoad();
+  }
+
+  @When("Open invalid path wrangler page")
+  public void openInvalidPathWranglerPage() {
+    SeleniumDriver.openPage(Constants.BASE_URL + "/cdap/ns/default/wranglerInvalidPath/invalid-workspace-id");
```
Comment (radhikav1) [inline src/e2e-test/java/io/cdap/cdap/ui/stepsdesign/PageLevelErrors.java]: could use as  : Constants.BASE_STUDIO_URL + "wranglerInvalidPath/invalid-workspace-id"

File: src/e2e-test/java/io/cdap/cdap/ui/stepsdesign/PageLevelErrors.java [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+
+  @When("Open invalid path details page")
+  public void openInvalidPathDetailsPage() {
+    SeleniumDriver.openPage(Constants.BASE_URL + "/pipelines/ns/default/viewInvalidPipelineDetails/pipelineName");
+    WaitHelper.waitForPageToLoad();
+  }
+
+  @When("Open invalid pipeline path details page")
+  public void openInvalidPipelinePathDetailsPage() {
+    SeleniumDriver.openPage(Constants.BASE_URL + "/pipelines/ns/default/view/invalidPipelineName");
```
Comment (radhikav1) [inline src/e2e-test/java/io/cdap/cdap/ui/stepsdesign/PageLevelErrors.java]: Define a constant like  **PIPELINES_BASE_URL =  Constants.BASE_URL + "/pipelines/ns/default"** which could be re-used.

File: src/e2e-test/java/io/cdap/cdap/ui/stepsdesign/PageLevelErrors.java (hunk 2 above)
Comment (de-lan) [reply src/e2e-test/java/io/cdap/cdap/ui/stepsdesign/PageLevelErrors.java]: Thanks! updated

**Review by radhikav1 — APPROVED** [review]

File: src/e2e-test/java/io/cdap/cdap/ui/stepsdesign/PageLevelErrors.java (hunk 3 above)
Comment (de-lan) [reply src/e2e-test/java/io/cdap/cdap/ui/stepsdesign/PageLevelErrors.java]: updated

**Review by sumengwang — APPROVED** [review]

**Review by mrahanjam — APPROVED** [review]

**Pull request merged**